### PR TITLE
Support tomcat static cluster configuration

### PR DESCRIPTION
**Pull request by nikosev**
Add configuration options to setup tomcat in cluster mode using StaticMembershipInterceptor class.
More info about the configuration can be found in RCIAM-271 ticket.

### CODE REVIEW

File: roles/tomcat/defaults/main.yml [hunk 1]
```diff
@@ -29,3 +29,22 @@ tomcat_connectors:
     redirect_port: "8443"
     #scheme: "https"
     uri_encoding: "UTF-8"
+
+cluster_mode: False
```
Comment (NicolasLiampotis) [inline roles/tomcat/defaults/main.yml:33]: `tomcat_cluster_mode: no`

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
rename cluster

File: roles/tomcat/defaults/main.yml (hunk 1 above)
Comment (nikosev) [reply roles/tomcat/defaults/main.yml]: done

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**